{{ cookiecutter.hosting_github_repo_name }}/SETUP/README.md

```diff
@@ -183,7 +183,9 @@ Once your repository, infrastructure, and web application have been set up, you
 
 Navigate and follow the instructions in the the file: `python/cdp_{{ cookiecutter.python_municipality_slug }}_backend/scraper.py`.
 
-As soon as you push your updates to your event gather function (`get_events`) to your GitHub repository, everything will be tested and configured for the next pipeline run.
+As soon as you push your updates to your event gather function (`get_events`) to your GitHub repository, everything will be tested and configured for the next pipeline run. Events are gathered from this function every 6 hours from the default branch via a Github Action cron job. If you'd like to manually run event gathering, you can do so from within the Actions tab of your repo -> Event Gather -> Run workflow.
+
+It is expected that the Event Index workflow will fail to start, as your database will not yet be populated with events to index.
```

Comment:
Note for you, I also just reduced the default cron schedule for indexing from three times a week to just once because it made little sense how much processing time was spent reindexing the whole set when only three or so new meetings were happening any given week. https://github.com/CouncilDataProject/cookiecutter-cdp-deployment/commit/3022746f7eaf82b3d38c2a6202d31b67aaa234f6

Would love your input on how a cookiecutter upgrade goes to pull in this minor change: https://github.com/smai-f/missoula-council-data-project/blob/main/admin-docs/updating-to-new-cookiecutter-releases.md